lib/restforce/concerns/composite_api.rb

```diff
@@ -29,7 +29,7 @@ def composite(all_or_none: false, collate_subrequests: false)
         if all_or_none && has_errors
           last_error_index = results.rindex { |result| result['httpStatusCode'] != 412 }
           last_error = results[last_error_index]
-          raise CompositeAPIError, last_error['body'][0]['errorCode']
+          raise CompositeAPIError.new(last_error['body'][0]['errorCode'], response)
```

Comment:
Should this be `response` or `results`?

Reply:
this should be Faraday's `response` because it is inheriting Faraday::ClientError. [reference Faraday's code](https://github.com/lostisland/faraday/blob/main/lib/faraday/error.rb#L32)